src/com/facebook/buck/android/dalvik/DalvikStatsCache.java

```diff
@@ -32,9 +32,12 @@
   }
 
   DalvikStatsTool.Stats getStats(FileLike entry) {
-    String name = entry.getRelativePath();
-    if (!name.endsWith(".class")) {
+    String[] pathParts = entry.getRelativePath().split("/");
+    String name = pathParts[pathParts.length-1];
+    if (!name.endsWith(".class") || name.equals("module-info.class")) {
       // Probably something like a pom.properties file in a JAR: this does not contribute
+      
+       //skipping special class files like module descriptor - here no classes will be declared and class visitor thows error
       // to the linear alloc size, so return zero.
```

Comment:
As I mentioned in the previous conversations, these comment should look like:
```
      // Probably something like a pom.properties file in a JAR: this does not contribute
      // to the linear alloc size, so return zero.
      // skipping special class files like module descriptor - here no classes will be
      // declared and class visitor throws error.
```

Reply:
sorry. Updated it.